source/User_Guide/Transactional_Templates/how_to_send_an_email_with_dynamic_transactional_templates.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: page
+weight: 100
+title: How to send an email with Dynamic Transactional Templates
+navigation:
+  show: false
+seo:
+  title: How to send an email with Dynamic Transactional Templates
+  override: true
+  description: 
+---
+
+- [Before you begin](#-Before-you-begin)
+- [Design a dynamic transactional template](#-Design-a-dynamic-transactional-template)
+- [Send a transactional email](#-Send-a-transactional-email)
+  - [Send email using the SendGrid API](#-Send-email-using-the-SendGrid-API)
+  - [Send email using SMTP](#-Send-email-using-SMTP)
+
+{% anchor h2 %}
+Before you begin
+{% endanchor %}
+
+Before you create and send an email using a dynamic transactional template you need to do the following:
+
+* Create a SendGrid account
+* Create an [API Key]({{root_url}}/User_Guide/Settings/api_keys.html)
+* Add an unsubscribe group (optional)
+
+{% anchor h2 %}
+Design a dynamic template
+{% endanchor %}
+
+1. Open the [transactional templates](https://sendgrid.com/dynamic_templates) page and click **Create Template**.
+1. Add a unique template name and then click **Save**. 
+1. To begin editing your new template, click **Add Version**.
+1. Select an editor and click **Continue**.
+1. Design your template. For more information on designing a Dynamic Transactional Template using the Design Editor or Code Editor, see [Create and edit Dynamic Transactional Templates]({{root_url}}/User_Guide/Transactional_Templates/create_and_edit_dynamic_transactional_templates.html).
+
+{% anchor h2 %}
+Send a transactional email
+{% endanchor %}
+
+{% anchor h3 %}
+Send transactional email using the SendGrid API
+{% endanchor %}
+
+*To send a dynamic transactional email:*
+
+In order to send a dynamic transactional email using cURL, set your call up to look something like this:
+
+{% codeblock %}
+curl -X "POST" "https://api.sendgrid.com/v3/mail/send" \
+     -H 'Authorization: Bearer [YourAPIKeyGoesHere!]' \
+     -H 'Content-Type: application/json' \
+     -d $'{
+  "from": {
+    "email": "example@.sendgrid.net"
+  },
+  "personalizations": [{
+    "to": [{
+      "email": "example@sendgrid.net"
+    }],
+    "dynamic_template_data": {
+      "guest": "Anthony Wang"
+      "partysize": 4,
+      "english": true,
+      "date": "April 1st, 2018"
+    }
+  }],
+  "template_id": "[template_id]",
+}'
+{% endcodeblock %}
+
+It is important to note two sections of this call when using dynamic templates.
+
+In order to send dynamic content, you need to specify that content in a dynamic template data section.
```

Comment:
~~In order to send dynamic content, you need to specify that content in a dynamic template data section.~~

In order to send dynamic content, you need to specify a collection of key/value pairs following the pattern "substitution_tag":"value to substitute". All are assumed to be strings. These handlebar substitutions apply to the text and html content of the body of your email, in addition to the subject and reply-to parameters. The total collective size of your handle bar substitutions may not exceed 10,000 bytes per personalization object.

maxProperties: 10000